google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/SpannerRpcViews.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.spanner.spi.v1;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableList;
+import io.opencensus.stats.*;
+import io.opencensus.tags.TagKey;
+import io.opencensus.stats.Aggregation.Distribution;
+import io.opencensus.stats.Measure.MeasureLong;
+import io.opencensus.stats.Aggregation.Sum;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+@VisibleForTesting
+public class SpannerRpcViews {
+
+    /** Unit to represent milliseconds. */
+    private static final String MILLISECOND = "ms";
+    /** Unit to represent counts. */
+    private static final String COUNT = "1";
+
+    /** TagKeys */
+    public static final TagKey GRPC_CLIENT_METHOD = TagKey.create("grpc_client_method");
+    public static final TagKey SPANNER_PROJECT_ID = TagKey.create("spanner_project_id");
+    public static final TagKey SPANNER_INSTANCE_ID = TagKey.create("spanner_instance_id");
+    public static final TagKey SPANNER_DATABASE_ID = TagKey.create("spanner_database_id");
+
+    public static final StatsRecorder STATS_RECORDER = Stats.getStatsRecorder();
+    /**
+     * GFE t4t7 latency extracted from server-timing header.
+     * */
+    public static final MeasureLong SPANNER_GFE_LATENCY =
+            MeasureLong.create(
+                    "cloud.google.com/java/spanner/gfe_latency",
+                    "Latency between Google's network receives an RPC and reads back the first byte of the response",
```

Comment:
Maybe `Latency between Google's network receiving an RPC and reading back the first byte of the response` reads better?